boot/zephyr/include/sysflash/sysflash.h

```diff
@@ -5,9 +5,13 @@
  */
 
 #if USE_PARTITION_MANAGER
-/* Blocking the rest of the file */
-#define __SYSFLASH_H__
+/* blocking the rest of the file */
+#define __sysflash_h__
```

Comment:
Shouldn't that be in uppercase to make an effect?

Reply:
These two lines shouldn't be her at all, removed